react-list.es6

```diff
@@ -23,7 +23,22 @@ const SIZE_KEYS = {x: 'width', y: 'height'};
 
 const NOOP = () => {};
 
-const PASSIVE = {passive: true};
+// If a browser doesn't support the `options` argument to
+// add/removeEventListener, we need to check, otherwise we will
+// accidentally set `capture` with a truthy value.
+const PASSIVE = (() => {
+  if (!process.browser) return false;
```

Comment:
I'm not familiar with `process.browser`? AFAIK `process` isn't a standard global in the browser. Can we go with `typeof window === 'undefined'`?